runtime/src/dispatcher.rs

```diff
@@ -53,7 +53,8 @@ use crate::{
 };
 
 /// Maximum amount of requests that can be in the dispatcher queue.
-const BACKLOG_SIZE: usize = 1000;
+// XXX: make this configurable?
+const BACKLOG_SIZE: usize = 10000;
```

Comment:
Is this related to https://github.com/oasisprotocol/oasis-core/issues/2548?

Reply:
Right, yeah we should probably do #2548 now that CheckTx is required on executors